bootloader/src/pyi_archive.c

```diff
@@ -333,6 +333,9 @@ int pyi_arch_open(ARCHIVE_STATUS *status)
 #elif defined(__APPLE__)
     int i;
     int alignment = 4096;
+#else
```

Comment:
This is duplicate. Simply remove the __APPLE__ line, since the part which contains the same.